props/prop_base.h

```diff
@@ -866,6 +878,23 @@ class PropBase : CommandParser, Looper, protected SaberBase {
   uint32_t last_on_time_;
 #endif
 
+#ifdef ENABLE_MENU_DIMBLADE
+  bool change_brightness_ = false;
+  bool BrightnessChangeMode(){return change_brightness_;}
+  void ToggleBrightnessChangeMode() {
```

Comment:
This change doesn't seem to add a way to actually access this menu, except from the serial monitor.

Reply:
Please look at my earlier comment own how I would like to proceed with implementing multiple menus, and the way they are accessed.

https://github.com/profezzorn/ProffieOS/pull/88#discussion_r484763570